CorsixTH/Lua/cheats.lua

```diff
@@ -83,7 +83,10 @@ function Cheats:cheatResearch()
 end
 
 function Cheats:cheatEmergency()
-  if not self.hospital:createEmergency() then
+  local outcome, err = self.hospital:createEmergency()
```

Comment:
It's nicer if `err` always contains the reason (and is `nil` if there is no error, probably), and then you have no need for `outcome`